python/src/main/resources/grpc/python/ipython_client.py

```diff
@@ -23,14 +23,23 @@
 def run():
     channel = grpc.insecure_channel('localhost:50053')
     stub = ipython_pb2_grpc.IPythonStub(channel)
-    response = stub.execute(ipython_pb2.ExecuteRequest(code="import time\nfor i in range(1,4):\n\ttime.sleep(1)\n\tprint(i)\n" +
-                                                            "%matplotlib inline\nimport matplotlib.pyplot as plt\ndata=[1,1,2,3,4]\nplt.figure()\nplt.plot(data)"))
+    response = stub.execute(ipython_pb2.ExecuteRequest(code="""
+    
+from bokeh.io import output_notebook
+import bkzep
+output_notebook(notebook_type='zeppelin')
+
+import hvplot.streamz
+from streamz.dataframe import Random
```

Comment:
doesn't these pull in bokeh and hvplot as hard dependencies for this? can we do this lazily or optionally?

Reply:
This is just for my local test, I will revert these changes